event/datadeal/data_delivery_event.go

```diff
@@ -0,0 +1,231 @@
+package datadeal
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	datadealtypes "github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/medibloc/panacea-doracle/config"
+	"github.com/medibloc/panacea-doracle/crypto"
+	"github.com/medibloc/panacea-doracle/event"
+	"github.com/medibloc/panacea-doracle/panacea"
+	log "github.com/sirupsen/logrus"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+type DataDeliveryVoteEvent struct {
+	reactor event.Reactor
+}
+
+func NewDataDeliveryVoteEvent(s event.Reactor) DataDeliveryVoteEvent {
+	return DataDeliveryVoteEvent{s}
+}
+
+func (e DataDeliveryVoteEvent) GetEventType() string {
+	return "data_delivery"
+}
+
+func (e DataDeliveryVoteEvent) GetEventAttributeKey() string {
+	return "vote_status"
+}
+
+func (e DataDeliveryVoteEvent) GetEventAttributeValue() string {
+	return "'started'"
+}
+
+func (e DataDeliveryVoteEvent) EventHandler(event ctypes.ResultEvent) error {
```

Comment:
IMO, We need to print a log when execute transaction.
Because the way to know how and which requested was voted for can be found in Oracle's log.

Reply:
I think it is necessary. But I'm not sure how to get the tx info from the event. Do you know how to get it?

Reply:
Sorry, I wrote the wrong comment.
I think it would be good print before transaction. :)
So we can use msgVoteDataDelivery object.

if the transaction failed, we need to know the request information. (ref: https://github.com/medibloc/panacea-doracle/pull/91/files#diff-0c6d9bfa1427c39a9489d371dd13d4d5f847f7a562817ee976de39cefadf3bc1R87)